config.example.yml

```diff
@@ -11,8 +11,10 @@ cpus: 2
 # - shortname: The site's shortname.
```

Comment:
Ah, I see. Needs to be separate because sites' docroots are handled differently. Cool!